packages/api/cms-api/src/mikro-orm/migrations/Migration20240702123233.ts

```diff
@@ -25,6 +25,11 @@ export class Migration20240702123233 extends Migration {
         const mimetypeToExtensionsMap: { [key: string]: string[] } = mimetypes.reduce((prev, m) => {
             const mimetype = m.mimetype;
             const extensions = getValidExtensionsForMimetype(mimetype);
+
+            if(extensions === undefined || extensions.length === 0) {
+                throw new Error(`No valid extensions found for mimetype ${mimetype}`);
```

Comment:
Maybe we should include in the message that the mimetype isn't allowed in the DAM and the file should be removed. What do you think?

Reply:
What about files with extensions and mime types that mime-db doesn't (yet) know about? Could we allow this by providing an extension before running this migration?

Like: "To fix this, either remove the file or add the extension to the file name and try to run the migration again"

Or should we neglect this and always suggest removing the file?

Reply:
> Could we allow this by providing an extension before running this migration?

Theoretically we could. However, we can't verify if the extension is correct. 

> Or should we neglect this and always suggest removing the file?

I would do this. 

When uploading files to the DAM, the mimetype is verified against mime-db too. You can only upload a file with a mimetype that is 

- allowed
- known to mime-db (except `application/x-zip-compressed` which we explicitly allowed)

So normally a file with an unknown mimetype can't be uploaded to the DAM anyway. 

Up until now, all files that caused problems were added by a sync from the old CMS (that bypassed the validation). All of these files were of type `application/x-cfb` and weren't actually used on the site. So I argue it's not a real problem.